packages/wallet/puppeteer/__tests__/funding.test.ts

```diff
@@ -11,7 +11,7 @@ import {
 } from "../helpers";
 import Emittery from "emittery";
 
-jest.setTimeout(30000);
+jest.setTimeout(60000);
```

Comment:
Is there any way to speed these up?

Reply:
Probably!

But currently the `puppeteer` tests seems to be taking around a minute in total for the whole test run (and is parallelized with other build steps) so I don't think it's worth the effort to try and get them faster, since they're not the slowest step.